README.md

```diff
@@ -44,6 +46,57 @@ node            |  |                               |  |
     make install
     ```
 
+### Pre-built binary
+
+If you'd rather not install from source, you can download a pre-built binary from the [releases](https://github.com/celestiaorg/celestia-app/releases) page.
+
+1. Navigate to the latest release on <https://github.com/celestiaorg/celestia-app/releases>.
+1. Download the binary for your platform (e.g. `celestia-app_Linux_x86_64.tar.gz`) from the **Assets** section.
+1. Extract the archive
+
+    ```shell
+    tar -xvf celestia-app_Linux_x86_64.tar.gz
+    ```
+
+1. Verify the extracted binary works
+
+    ```shell
+    ./celestia-appd --help
+    ```
+
+#### Optional: Verify the pre-built binary checksums and signatures
+
+If you use a pre-built binary, you may also want to verify the checksums and signatures.
+
+1. Navigate to the latest release on <https://github.com/celestiaorg/celestia-app/releases>.
+1. Download `checksums.txt`, `checksums.txt.sig`, and the binary for your platform (e.g. `celestia-app_Linux_x86_64.tar.gz`) from the **Assets** section.
+1. Verify the checksums
+
+    ```shell
+    sha256sum --ignore-missing --check checksums.txt
+    ```
+
+    You should see output like this:
+
+    ```shell
+    celestia-app_Linux_x86_64.tar.gz: OK
+    ```
+
+1. Clone this repo
+1. Verify the signature via the [verify-signature.sh](./scripts/signing/verify-signature.sh) script.
+
+    ```shell
+    ./scripts/signing/verify-signature.sh checksums.txt.sig checksums.txt
+    ```
```

Comment:
Tested this again with a pre-built binary that I built on a Digital Ocean droplet. Full output:

```
root@rootulp-goreleaser:~/celestia-app# ./scripts/signing/verify-signature.sh build/goreleaser/checksums.txt.sig build/goreleaser/checksums.txt
Importing the public keys in /root/celestia-app/scripts/signing/keys
Importing /root/celestia-app/scripts/signing/keys/celestia-app-maintainers.asc
gpg: key D469F859693DC3FA: "celestia-app-maintainers <celestia-app-maintainers@celestia.org>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1
Imported /root/celestia-app/scripts/signing/keys/celestia-app-maintainers.asc
Importing /root/celestia-app/scripts/signing/keys/test-goreleaser-signing-2.asc
gpg: key A6919B058B94096D: "test-goreleaser-signing-2 <test-goreleaser-signing-2@gmail.com>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1
Imported /root/celestia-app/scripts/signing/keys/test-goreleaser-signing-2.asc
Verifying the signature of build/goreleaser/checksums.txt.sig with build/goreleaser/checksums.txt
gpg: Signature made Thu Oct  5 21:55:19 2023 UTC
gpg:                using RSA key 064F5253EBF7AA82C9A7CFE7A6919B058B94096D
gpg: Good signature from "test-goreleaser-signing-2 <test-goreleaser-signing-2@gmail.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 064F 5253 EBF7 AA82 C9A7  CFE7 A691 9B05 8B94 096D
```